### PR TITLE
Css changed

### PR DESCRIPTION
**Pull request by Arijitnaskar12**

### CODE REVIEW

File: style.css
Comment (AvinashP-1611) [inline style.css:1]: Ok Arijit good job

**Review by AvinashP-1611 — CHANGES_REQUESTED** [review]
Arijit do more changes****

File: style.css
Comment (Arijitnaskar12) [reply style.css:1]: Thank you sir

Comment (Arijitnaskar12) [reply style.css:1]: What changes should i do?

**Pull request merged**